### PR TITLE
Avoid COCO dataset download

### PR DESCRIPTION
**Pull request by MohitIntel**
Avoids re-download of datasets and prevents hangs in CI farm.
Introduces a new env variable DATA_CACHE for specifying path of local dataset.

### CODE REVIEW

**Review by jiminha — APPROVED** [review]
LGTM!

File: tests/test_examples.py [hunk 1]
```diff
@@ -575,6 +581,7 @@ class MultiCardSpeechRecognitionExampleTester(
     ExampleTesterBase, metaclass=ExampleTestMeta, example_name="run_speech_recognition_ctc", multi_card=True
 ):
     TASK_NAME = "regisss/librispeech_asr_for_optimum_habana_ci"
+    DATASET_NAME = os.environ.get("DATA_CACHE", 0)
```
Comment (jiminha) [inline tests/test_examples.py:584]: what happen if you don't set this environment?

Comment (MohitIntel) [reply tests/test_examples.py:584]: Verified working even without DATA_CACHE on a different machine. Both clip-roberta and speech-wav2vec2 work fine.

**Comment by regisss** [comment]
@MohitIntel I guess we can close this one since #848 got merged?

**Pull request closed without merging**